lib/api/core/httpRouter/index.js

```diff
@@ -52,11 +52,16 @@ class Router {
 
     this.defaultHeaders = {
       'content-type': 'application/json',
+      'Accept-Encoding': 'identity',
       'Access-Control-Allow-Origin': kuzzle.config.http.accessControlAllowOrigin || '*',
       'Access-Control-Allow-Methods': 'GET,POST,PUT,DELETE,OPTIONS,HEAD',
-      'Access-Control-Allow-Headers': 'Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With'
+      'Access-Control-Allow-Headers': 'Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With, Content-Encoding'
```

Comment:
not directly related with the issue this PR resolves... but with @xbill82 we noticed that we need also to add `x-kuzzle-volatile` and  `Content-Length` to allow-headers. (see #1135 )

Reply:
Of course!
Headers updated :+1: